lib/feature/wallet/widgets/account_asset_tab/ton_wallet_asset/ton_wallet_asset_widget.dart

```diff
@@ -25,6 +25,7 @@ class TonWalletAssetWidget extends StatelessWidget {
         balanceService: inject<BalanceService>(),
         currencyConvertService: inject<CurrencyConvertService>(),
         nekotonRepository: inject<NekotonRepository>(),
+        balanceStorage: inject(),
```

Comment:
Type omitted, is it intentional?

Reply:
Same, for dart since 2-nd version it's not necessary, especially if we have same registered type in getIt and expecting object